Moved pages/hr and contributions/hr to summary page for #2748

sylvieed opened the pull request:
_Resolves #2748_

Previously, the pages transcribed per hour and contributions per hour statistics as well as a table with more numbers were on their own "progress metrics" page accessed by clicking the "transcribed pages" statistic on the admin homepage.

![progress metrics page](https://user-images.githubusercontent.com/35716893/154184927-2d120c40-607a-4201-ab85-843ce0043ea0.jpg)

This pull moves these numbers (pages/hr and contributions/hr) to the beginning of the admin homepage.

![summary page](https://user-images.githubusercontent.com/35716893/154185585-aa183ce1-e87f-4ae7-9cf7-0a6139dc0a43.jpg)

I decided not to bring the table to the admin homepage as well because it made the page a bit too long and seemed out of place. So, it is now in the "progress metrics" page by itself.
![new progress metrics page](https://user-images.githubusercontent.com/35716893/154186053-db17946c-a597-40a0-a16f-913547d9f856.jpg)

benwbrum reviewed (changes requested):
The code looks good, but the layout could use improving.  How about moving the two speedometer statistics to the top line by themselves, then having the other items in the 2x4 table below them?

saracarl commented:
I like it!  Each line makes sense conceptually -- the speedometer bits,
then collections/works/articles/notes; owners/users.  Good job.

On Mon, Feb 28, 2022 at 9:48 AM Sylvie Daines ***@***.***>
wrote:

> How does this look @saracarl <https://github.com/saracarl> @benwbrum
> <https://github.com/benwbrum>?
>
> [image: new admin dashboard]
> <https://user-images.githubusercontent.com/35716893/156012788-67b600a5-1529-43d1-9acc-66120f94c055.jpg>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/benwbrum/fromthepage/pull/3007#issuecomment-1054392392>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFM7VW4MLCUNBGLIHEMB7DU5OKNHANCNFSM5OQKFHJA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Sara Brumfield
Partner, Brumfield Labs
Creators of FromThePage <http://www.fromthepage.com>
<https://www.fromthepage.com>

sylvieed commented:
OK, those changes are pushed to this branch. 

I moved the table to the summary page and changed it to use the `.admin-grid.datagrid.striped` layout like the owners tab.

Because there's nothing left in the `progress_metrics` page, I deleted it in `routes.rb`, `admin_controller.rb` and `progress_metrics.html.slim`

saracarl commented:
Whoa -- this got our tests up to 80%!!!
